src/components/events/CalendarEvents.js

```diff
@@ -78,6 +80,11 @@ const CalendarEvents = ({ limited = false }) => {
           events={events.filter((e) => e.start >= new Date())}
           size={3}
         />
+        <div className="mb-24">
```

Comment:
This "See Calendar" button is not going to be in the calendar events file. It should be in the home page. Check Figma.